styles.css

```diff
@@ -0,0 +1,3 @@
+.headings{
+    color: #0c6ae4;
+}
```

Comment:
**Linters tests are not passing ❌:**
- [ ] I suggest addressing those issues in your code to ensure that it is clean and well-formatted. This will not only make your code better and easier to read, but it will also help you become a better developer for future projects.
Please take some time to review the linter test results and fix any errors or warnings that are reported. If you need any assistance with this process, feel free to reach out for help.

![image](https://user-images.githubusercontent.com/107952724/234418501-b8d81715-fb56-411c-a3ad-07b4d28afb72.png)
_You can check the linter errors in the **Actions** section or inside your project's terminal by running this command: 
`npx stylelint "**/*.{css,scss}"`_